stix_shifter_modules/splunk/README.md

```diff
@@ -1,80 +1,675 @@
-# Splunk
+# Splunk Enterprise
 
 ## Supported STIX Mappings
 
-See the [table of mappings](splunk_supported_stix.md) for the STIX objects and operators supported by this connector.
```

Comment:
make sure you keep the link here for the table of mappings.

Reply:
added the link as suggested.